tests/consumers/test_consumer_builder.py

```diff
@@ -22,7 +32,7 @@
     kafka_params=KafkaParameters(
         raw_topic=None,
         replacements_topic=None,
-        bootstrap_servers=None,
+        bootstrap_servers=os.environ.get("DEFAULT_BROKERS", "localhost:9092"),
```

Comment:
Can we leave this as None? I think the test is failing currently because this is expecting an array and is getting a string. `None` means use the default which is probably ok for this test.

Reply:
Thanks for catching this!